modules/ROOT/pages/migration-from-6x.adoc

```diff
@@ -3,62 +3,310 @@
 
 [[plugins]]
 == Plugins
 
-...
+=== Changed
+
+[[The-list-plugin-is-now-required-to-be-added-to-TinyMCE-config-in-order-to-execute-list-commands]]
+==== The list plugin is now required to be added to {productname} config, in order to execute list commands
+
+A breaking change has been introduced regarding the insertion of **lists** using the `InsertOrderedList` and `InsertUnorderedList` commands.
+
+Despite the presence of these commands, they have been functional even without the `list` plugin being added to the {productname} init configuration.
+
+{productname} 7.0, now **requires** that the `list` plugin is present in the `init` configuration for executing these insert list commands.
+
+As a result, and users that notice styling issues with their existing documents will be required to add the `list` plugin to their `init` config, as executing the `InsertOrderedList` and `InsertUnorderedList` commands will have longer effect the editor content.
+
+.Example adding list plugin
+[source, js]
+----
+tinymce.init({
+  selector: "textarea",
+  plugins: [
+    "lists"
+  ],
+  toolbar: "bullist numlist",
+});
+----
+
 
-[[schema]]
-== Schema
+[[removed-plugins]]
+=== Removed Plugins
+
+In {productname} 7.0 the below plugin has been removed.
+
+[[removed-plugins-template-plugin]]
+==== Removed open-source `Template` plugin
+
+The open-source `Template` plugin and associated config options have been removed in {productname} 7.0.
+
+Customers using the `template` plugin need to migrate to the premium `advtemplate` which contains all the functionality except some esoteric options that was created very long ago. Instead of simply porting over these options, we will listen to customer feedback and quickly add the ones customer’s are relying on. 
+
+This plugin has been replaced with the **Advanced Template** plugin, see: xref:advanced-templates.adoc[Advanced Template] for more details.
+
+[IMPORTANT]
+For customers using the `template` plugin and are looking to migrate to the premium `advtemplate` plugin which contains all the functionality except some specific options which are noted below.
+
+Options not available in **Advanced Template** plugin
+    
+* `template_cdate_classes`
+* `template_cdate_format`
+* `template_mdate_classes`
+* `template_mdate_format`
+* `template_replace_values`
+* `template_preview_replace_values`
+* `template_selected_content_classes`
 
-...
 
 [[ui-and-ux-elements-and-components]]
 == UI and UX elements and components
 
-...
+=== Improved
+
+[[ui-and-ux-elements-and-components-improved-how-styles-are-applied-when-resizing-rows-and-columns-for-tables]]
+==== Improved how styles are applied when resizing rows and columns for Tables.
+
+Previously added numerous style elements when resizing table rows and columns such as on the `table element`, `tr elements`, and `td elements`. This resulted in unappealing HTML output.
```

Comment:
```suggestion
Previously added numerous style elements when resizing table rows and columns such as on the `table`, `tr`, and `td` elements. This resulted in unappealing HTML output.
```

Reply:
this was re-worded in a previous commit, and the context is outdated.